src/app/features/genes/components/gene-comparison-tool/gene-comparison-tool.component.ts

```diff
@@ -199,10 +199,10 @@ export class GeneComparisonToolComponent implements OnInit, AVI, OnDestroy {
       return gene.target_risk_score;
     }
     if (columnName === this.scoresColumns[1].field) {
-      return gene.multi_omics_score;
+      return gene.genetics_score;
     }
```

Comment:
I think something along those lines would be great!  However, for the moment I think it might be best to revisit this in the future since I want to change the collection schema in MongoDB.  Some of the columns are for tissues and some of them are for scores.  Since the scores are not alphabetically sorted and have been recently renamed, they have fallen out of sync with the collection naming.  It would be nice to use new properties in a custom class for scores columns that would make sorting them more elegant.

Reply:
Got it! That sounds like a great solution